src/libraries/Common/src/Interop/Windows/Crypt32/Interop.FindOidInfo.cs

```diff
@@ -75,28 +75,28 @@ internal static CRYPT_OID_INFO FindOidInfo(CryptOidInfoKeyType keyType, string k
                 if (!OidGroupWillNotUseActiveDirectory(group))
                 {
                     OidGroup localGroup = group | CRYPT_OID_DISABLE_SEARCH_DS_FLAG;
-                    IntPtr localOidInfo = CryptFindOIDInfo(keyType, rawKey, localGroup);
-                    if (localOidInfo != IntPtr.Zero)
+                    CRYPT_OID_INFO* localOidInfo = CryptFindOIDInfo(keyType, rawKey, localGroup);
+                    if (localOidInfo != null)
                     {
-                        return Marshal.PtrToStructure<CRYPT_OID_INFO>(localOidInfo);
+                        return *(CRYPT_OID_INFO*)localOidInfo;
```

Comment:
Why is a cast required in these returns?

Reply:
Not needed. I forgot to delete them when I have fixed the PInvoke signature to return the proper type.